scripts/release-build.test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const assert = require('assert');
+const { execSync } = require('child_process');
+
+const prebuiltPath = '../build';
+
+describe('release-build', () => {
+  it('pre-built bundles are identical to the source', done => {
```

Comment:
```suggestion
  it('pre-built bundles are identical to the source', () => {
```

This test is fully synchronous. That means you actually won't need the timeout, because it's impossible for mocha to run and deem the test timed out. It can mark it as slow in the output, though.

For tests that need to be async, mocha supports async functions.